internal/runner/jest_test.go

```diff
@@ -53,6 +53,8 @@ func TestNewJest(t *testing.T) {
 }
 
 func TestJestRun(t *testing.T) {
+	mockCwd(t, "./testdata/jest")
```

Comment:
This is required because Jest need to find `jest.config.json` which is now stored inside `testdata/jest` directory.